addons/mail/models/mail_template.py

```diff
@@ -160,6 +160,7 @@ def default_get(self, fields):
                                                   "destination document model (sub-model).")
     null_value = fields.Char('Default Value', help="Optional value to use if the target field is empty")
     copyvalue = fields.Char('Placeholder Expression', help="Final placeholder expression, to be copy-pasted in the desired template field.")
+    scheduled_date = fields.Char('Scheduled Date')
```

Comment:
I would add an help to explain the behavior set / not set (like on mail_mail) + tell that jinja placeholders may be used here.